.github/workflows/ci.yml

```diff
@@ -132,7 +132,10 @@ jobs:
       fail-fast: true
       matrix:
         python-version: ["3.7", "3.8", "3.9"]
-        nautobot-version: ["1.1.5", "1.2.11", "1.3.7"]
+        nautobot-version: ["1.3.8"]
```

Comment:
which was the problem with mysql?
BTW, would make sense to leave the nautboto-version to "stable" to always check against hte latest by default? (don't recall if we already commented this point)

Reply:
`stable` doesn't work. There is a parser error.

Reply:
I'm just trying to get things into a working state. What didn't work with MySQL is that I didn't have a matrix set up for it.

Reply:
sorry Josh, I don't get it, because the matrix was already set up before, no?

Reply:
Not for `stable`. It was set up to test individual versions.

Reply:
yes, I understand, not for stable, but the matrix was working well for mysql before, no?

Reply:
Yes it was. I guess the wording of `fix mysql` was poor. Probably should have just been `update mysql matrix`.